### DISCUSSION TITLE
Create Simulation

### DISCUSSION THREAD

**Discussion by TitusRoy** [Open-Systems-Pharmacology/Forum#1123] [Questions & Problems]
After done with Building blocks, I started with create simulation. 

I gave all the input parameters for model structure, compounds, processes, administrations, events. But I didn't add observers in observers tab, I couldn't finish it by clicking "OK". The OK button is not ON. 

Where I am going wrong? Is it compulsory to add any one of the observers? 

KR

**Comment by msevestre** [comment]
Can you make a screenshot?
Did you enter a name for the simulation? (see the red cross at the top)

**Comment by TitusRoy** [comment]
I have attached the screenshot for your reference.

Kindly find it

**Comment by msevestre** [comment]
you need to attach it on github. Attachments are removed via emails

**Comment by TitusRoy** [comment]
![20220207_200257](https://user-images.githubusercontent.com/99109232/152811294-699ae8c9-831c-4c41-9c93-8db78329f135.jpg)

**Comment by msevestre** [comment]
Can you make a screenshot of all tabs? This is very weird indeed

**Comment by TitusRoy** [comment]
![20220207_203112](https://user-images.githubusercontent.com/99109232/152813662-4e01a2e6-ea0b-45d5-b281-e0507047008d.jpg)
![20220207_203150](https://user-images.githubusercontent.com/99109232/152813713-150b5124-4fcb-42fb-8b21-939f4d0ff918.jpg)
![20220207_203200](https://user-images.githubusercontent.com/99109232/152813758-bc9d13e3-d48d-4657-9e3f-cecece32a3fa.jpg)
![20220207_203208](https://user-images.githubusercontent.com/99109232/152813781-3b4f37ad-54d9-4528-99e3-e8878aabfb2c.jpg)
![20220207_203052](https://user-images.githubusercontent.com/99109232/152813801-2903482e-0d02-411a-b62a-94e20d06a62a.jpg)

**Comment by msevestre** [comment]
A formulation is missing for your oral absorption

**Comment by TitusRoy** [comment]
Thank you so much.

It worked.


Best Regards,
TRoy.
